### DISCUSSION TITLE
Remove additional tags.

### DISCUSSION THREAD

**Discussion by drewen1** [fo-dicom/fo-dicom#1701] [Q&A]
After use fo-dicom to edit some tags, there are some additional tags has been added to the dicom file (Ex. 0002, XXXX). Is there a way to do not add these tags after edit the file? 
![additional tags](https://github.com/fo-dicom/fo-dicom/assets/22739159/daaa45c6-ddfa-4fae-abdf-8f13de5beb6e)

**Comment by mrbean-bremen** [comment]
These (tags in group 2) are meta tags that are always created or adapted if transferring, saving or otherwise manipulating a DICOM dataset. They are not directly related to the content, only to the encoding of the information, and should usually be ignored.
The actual content starts only with tags in group 8.

If you really want to remove these, you can do this by filtering them:
```cs
dataset.Remove(x => x.Tag.Group == 2);
```

**Reply by drewen1** [reply]
Thank you for the reply. I have tried to use this code but seems it will add the meta info automatically.

**Reply by mrbean-bremen** [reply]
Well, as I wrote: the meta tags are added if  if transferring or saving a DICOM file, and they are mandatory.
What is your actual use case for removing them? What do you mean by "it" in " it will add the meta info automatically"?

**Reply by drewen1** [reply]
"it" means fo-DICOM library. The use case is that our program generated no include meta info DICOM files, I need anonymize these DICOM files, but after use fo-DICOM to anonymize the file, fo-DICOM will add meta info automatically. Then our program cannot import these anonymized DICOM files due to it has meta info header.

**Reply by mrbean-bremen** [reply]
You mean you want to write DICOM files without the metainfo? These would not be valid files, you need to have at least the transfer syntax and the SOP class for reading the file. I probably didn't understand what you are doing.

**Reply by drewen1** [reply]
I agree with you. The meta info is necessary. I will try to figure out the problem in our program. Thank you again.
